src/Widgets/DataLoadMoreButtonWidget/DataLoadMoreButtonWidgetComponent.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  WidgetTag,
+  isComparisonActive,
+  isInPlaceEditingActive,
+  provideComponent,
+  useData,
+} from 'scrivito'
+import { DataLoadMoreButtonWidget } from './DataLoadMoreButtonWidgetClass'
+import { useContext } from 'react'
+import { DataScopeParamsContext } from '../../Components/DataScopeParamsContext'
+import { buttonSizeClassName } from '../../utils/buttonSizeClassName'
+import { alignmentClassNameWithBlock } from '../../utils/alignmentClassName'
+
+provideComponent(DataLoadMoreButtonWidget, ({ widget }) => {
+  const dataScope = useData()
+  const { limit, setLimit } = useContext(DataScopeParamsContext)
+
+  const moreAvailable =
+    limit < dataScope.transform({ limit: limit + 1 }).take().length
+
+  const classNames = ['btn']
+
+  const buttonSize = buttonSizeClassName(widget.get('buttonSize'))
+  if (buttonSize) classNames.push(buttonSize)
+
+  const buttonColor = widget.get('buttonColor') || 'btn-outline-primary'
+  if (buttonColor) classNames.push(buttonColor)
+
+  const alignmentClassName = alignmentClassNameWithBlock(
+    widget.get('alignment'),
+  )
+
+  if (!moreAvailable) {
+    if (!isInPlaceEditingActive() && !isComparisonActive()) return null
+
+    classNames.push('disabled')
```

Comment:
* [ ] As the only thing we want to indicate is that the button is conditional `disabled` is misleading.
* [ ] FYI I see that we can't go full WYSIWYG here but how about this: Make the button behave as intended for all pages but page 1.

Alternatively we could make a greyed-out paging button the default behavior except for cases when there's only one page. A disabled button on the last page could be the better UX as it more clearly indicates "you have successfully reached the last page".

Reply:
I removed `classNames.push('disabled')` but kept the grey-out stuff. The button will disappear if there is not more to load in visitor mode.